pyproject.toml

```diff
@@ -43,23 +43,34 @@ classifiers = [
 # the commit fail because "files were modified by this hook". In that case,
 # doing again the commit including changes in docs/requirements.txt will succeed.
 python = "^3.7"
-numpy = "*"
-scipy = "^1.2"
-pandas = "^1"
+numpy = [
+    {version = "^1.16.0", python = "^3.7"},
+    {version = "^1.18.0", python = "^3.8"}
+]
+scipy = [
+    {version = "^1.3.0", python = "^3.7"},
+    {version = "^1.4.0", python = "^3.8"}
+]
+pandas = "^1.1.0"
```

Comment:
I had forgotten we need such a recent version of pandas.
It convinces me even more that we should not bother with too old versions for other packages.